src/async-components/views/dialogs/eventindex/ManageEventIndexDialog.js

```diff
@@ -83,14 +92,18 @@ export default class ManageEventIndexDialog extends React.Component {
         const eventIndex = EventIndexPeg.get();
 
         if (eventIndex !== null) {
-            eventIndex.on("changedCheckpoint", this.updateCurrentRoom.bind(this));
+            eventIndex.on("changedCheckpoint", this.updateCurrentRoom);
+
+            try {
+                const stats = await eventIndex.getStats();
+                eventIndexSize = stats.size;
+                eventCount = stats.eventCount;
+            } catch {
+            }
```

Comment:
This needs a comment to explain why we're swallowing the exception without so much as logging it.